core/request.py

```diff
@@ -105,6 +105,16 @@ def __init__(self, request: webob.Request, response: webob.Response):
         self.is_deferred = False
         db.currentDbAccessLog.set(set())
 
+    def __getattr__(self, attr):
+        if attr == "isDevServer":
+            import warnings
+            msg = f"Use of `{attr}` is deprecated; Use conf[\"viur.project.is_dev_server\"] instead!"
+            warnings.warn(msg, DeprecationWarning, stacklevel=2)
+            logging.warning(msg)
+            return conf["viur.project.is_dev_server"]
+
+        return super().__getattr__(attr)
```

Comment:
We have properties for this in python!

```suggestion
    @property
    def isDevServer(self) -> bool:
          import warnings
          msg = f"Use of `{attr}` is deprecated; Use conf[\"viur.project.is_dev_server\"] instead!"
          warnings.warn(msg, DeprecationWarning, stacklevel=2)
          logging.warning(msg)
          return conf["viur.project.is_dev_server"]
```